packages/windows/vcruntime/vs2015/readme.txt

```diff
@@ -0,0 +1,15 @@
+Subject to Microsoft Visual Studio Community 2015 EULA [1], This package
+contains Visual C++ Runtime files as redistributable code:
+
+[1] https://visualstudio.microsoft.com/license-terms/microsoft-visual-studio-community-2015/

```

Comment:
I think this link is broken.
I tried to display this page in the browser(Firefox). However, I could not display this page.

Reply:
it seems that maybe  bottom language setting is set to "Japanese" for mirosoft.com.

Reply:
Oh.
Thank you for your information.
I could display this page when I changed the language setting to "English".